e2e/mock-api/src/app/app.certs.js

```diff
@@ -9,15 +9,15 @@
  */
 
 import { readFileSync } from 'fs';
-import { fileURLToPath } from 'url';
 import path from 'path';
 
-const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(__filename);
 
-const cert = readFileSync(path.resolve(__dirname, '../../../../node_modules/lws/ssl/lws-cert.pem'));
+const cert = readFileSync(
```

Comment:
I can work on getting esmodules to work but i also just felt this was easier. i had a require issue within an nx package when using esmodules